crates/typst/src/visualize/paint.rs

```diff
@@ -1,48 +1,59 @@
-use std::fmt::{self, Debug, Formatter};
-
 use ecow::EcoString;
 
-use crate::foundations::{cast, Repr};
-use crate::visualize::{Color, Gradient, GradientRelative};
+use crate::foundations::{cast, Repr, Smart};
+use crate::visualize::{Color, Gradient, RelativeTo};
+
+use super::Pattern;
```

Comment:
```suggestion
use crate::visualize::{Color, Gradient, Pattern, RelativeTo};
```
the `super` imports have been abolished in favor of `crate` imports to keep things more consistent across the codebase.